lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -53,6 +53,9 @@
 <!-- FAIL(errors-in-console): exception thrown -->
 <script type="text/javascript">throw new Error('An error');</script>
 
+<!-- Add non-functional script to appear to be a Wordpress page for Stack Packs. -->
+<script type="example" src="fake-script-wp-includes"></script>
```

Comment:
since we're adding this here, mind adding it to the smoke test?

Should just need to add something like
```js
artifacts: {
  Stacks: [{
    id: 'jquery',
  }, {
    id: 'wordpress',
  }],
},
```
in parallel to the `lhr` entry in `dbw-expectations.js`